### PR TITLE
Update FIPS algorithm list.

### PR DESCRIPTION
**Pull request by paulidale**
Some additional algorithms have been added to the FIPS validation.  Reflect this
in the appendix.

### CODE REVIEW

**Comment by paulidale** [comment]
The current list is causing some consternation about what is being included in the validation and what is missing.

File: docs/OpenSSL300Design.md [hunk 1]
```diff
@@ -2864,11 +2888,23 @@ Security Policy statement regarding the <a href="https://csrc.nist.gov/publicati
   <tr>
    <td>
    </td>
+  <tr>
```
Comment (t8m) [inline docs/OpenSSL300Design.md]: You seem to be inserting this hunk at a wrong place

Comment (paulidale) [reply docs/OpenSSL300Design.md]: Good point.  Editing this file is nasty.

Comment (paulidale) [reply docs/OpenSSL300Design.md]: Fixed now.

**Comment by mattcaswell** [comment]
Not really sure what the "compliant" column means any more. Wasn't it a "point in time" thing to indicate what algorithms where compliant with the standards at the time we originally drew up the list. Should we just drop that column now?

**Comment by paulidale** [comment]
The compliant column is not terribly useful.  This document is dated in other ways too.

People are looking at it and making false assumptions about what is going into the FIPS provider.  I'm aiming to avoid this confusion.

**Review by mattcaswell — APPROVED** [review]
I'm ok with this. I'd also be ok with a change that additionally dropped the compliant column.

**Comment by paulidale** [comment]
See #205 for the _compliance_ column removal and a bit of other clean up.

**Pull request closed without merging**